crypto/fipsmodule/cipher/cipher.c

```diff
@@ -255,6 +256,7 @@ static int block_remainder(const EVP_CIPHER_CTX *ctx, int len) {
 
 int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, uint8_t *out, int *out_len,
                       const uint8_t *in, int in_len) {
+  GUARD_PTR(ctx);
```

Comment:
Why there are no guards for CipherInit_ex, CipherUpdate and CipherFinal_ex? `ctx` is accessed there right away.